code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -62,6 +63,10 @@
 
 	var/obj/item/clothing/accessory/petcollar/collar = null
 	var/can_collar = 0 // can add collar to mob or not
+	var/can_trophy = 0
+	var/trophied = 0
+	var/obj/item/weapon/trophy/trophy = null
+	var/trophy_id = "generic" //for name and desc modification

```

Comment:
unnecessary with the trophy being able to be specified now--this should only be on the trophy itself (or not have an id at all and merely go by the trophy's icon state.